shared/tinyusb/mp_cdc_common.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2022 Blake W. Felt & Angus Gratton
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+#ifndef MICROPY_INCLUDED_SHARED_TINYUSB_MP_CDC_COMMON_H
+#define MICROPY_INCLUDED_SHARED_TINYUSB_MP_CDC_COMMON_H
+
+void tud_cdc_poll_interfaces(void);
```

Comment:
I know you didn't choose these names, but this might be a good time to clean this up. `tud_` is the prefix used by the TinyUSB device API itself.

What about `mp_usbd_cdc_poll_interfaces()`, and `mp_usbd_cdc.h` for the header & source file names?

If that's too verbose then we could go for a prefix of `mp_cdc...` in both places, instead?

Reply:
Hi Angus, thanks that's a good catch - I kind of did choose this name / add the `tud_` myself but you're right, that's a tinyusb header, we should definitely go with a mp header. I think verbose is fine here, it's not going to push out line lengths crazy in any places it's used.

FYI this change is a precurser to the main thing I've been working on in this space - getting the microypthon banner to be buffered at startup / printed on initial connection of mpremote, just like stm port does. This change is in final stages of testing now and should be pushed soon....

Reply:
Ooh, that'll be nice to have!

Reply:
The file / function names have been updated now in latest push.